src/external/index.js

```diff
@@ -1,9 +0,0 @@
-// import reactselect from './reactselect.scss';
-import reactdatepicker from './reactdatepicker.scss';
-
-const style = {
-  // reactselect,
-  reactdatepicker,
-};
```

Comment:
need to check that we're not using any of this in SS

Reply:
Checked, unused

Reply:
All of the datepickers look kosher in ss?